ABCD/data_convert.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+
+
+def get_args():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
```

Comment:
I don't see a `__doc__` function anywhere. Or does this double underscore method do something I'm not aware of?

Reply:
You're right! It should include __doc__ section for this to work. I could copy-paste contents of the README into __doc__ function? That way users can access it as command line help if they wish

Reply:
I typically put just a short description of the script in the argparse `description`. It populates into the help text every time you run with `-h` or`--help`.